pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -331,7 +331,7 @@ self: super: {
   pipes-websockets = dontCheck super.pipes-websockets;
   posix-pty = dontCheck super.posix-pty; # https://github.com/merijn/posix-pty/issues/12
   postgresql-binary = dontCheck super.postgresql-binary; # needs a running postgresql server
-  postgresql-simple-migration = dontCheck super.postgresql-simple-migration;
+  postgresql-simple-migration = doJailbreak (dontCheck super.postgresql-simple-migration);
```

Comment:
Is there an issue / fix for the version constraints for `postgresql-simple-migration` upstream?